inc/Editor/BlockManagement/BlockRegistration.php

```diff
@@ -77,6 +78,7 @@ public static function register_blocks(): void {
 				'availableBindings' => $available_bindings,
 				'loop' => $config['loop'],
 				'name' => $block_name,
+				'dataSource' => ConfigStore::get_data_source( $block_name ),
```

Comment:
Is this trying to describe the type of data source, like Airtable or Shopify? Maybe we can name it `dataSourceType`